definitions/src/asm.rs

```diff
@@ -147,6 +147,7 @@ impl AsmCoreMachine {
             machine.chaos_mode = 0;
         }
         machine.chaos_seed = 0;
```

Comment:
We can now remove those lines that set a certain field to 0

Reply:
Explicitly initializing these values helps reduce unexpected bugs, I think the code can be preserved.

Reply:
I disagree, the assumptions already changed when you do a `alloc_zeroed` at first. I think they would probably contribute to future confusions (what if `alloc_zeroed` is reverted back to `alloc`, and someone forgets to zero-initialize a newly added field, if none of the values has been manually set to zero, this mistake will be easily noticed)

Reply:
Agree with you. This part of the duplicate code has been removed.